samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -471,6 +471,9 @@ Testing
    Images in the testing section are generated for the debug device model registered by Nordic Semiconductor in the Google Nearby Console (see :ref:`fast_pair_locator_tag_google_device_model`).
    The debug device model name is covered by asterisks and the default Fast Pair logo is displayed instead of the one specified during the device model registration.
 
+   If the test Android device is set up for the primary email account that is not part of the Google email allow list for the FMDN feature, the following testing steps will fail at the FMDN provisioning stage for the debug (uncertified) device model model used by default in this sample.
```

Comment:
Can we move it to requirements section (user should be aware of it early I think) and rephrase it a bit? Something like:
```
By default the sample uses a debug Fast Pair Model ID.
If you want to test FMDN with a debug Model ID, primary email account used by your Android phone must be allowlisted by Google.
Use `Fast Pair Device Proposal Form`_ from Google to register your development email for the allow list.
```

Reply:
this is up to @annwoj, @divipillai. I can move it there

Reply:
I think it may be reasonable to let user know about the limitation early to avoid disappointment on test start

Reply:
I think we can keep this here as this is not breaking the user flow. The user just needs to register and then he can continue with the testing.